integrations/segment/gitbook-manifest.yaml

```diff
@@ -41,6 +43,11 @@ configurations:
                 type: string
                 title: Write Key
                 description: Key provided by Segment to identify the data source.
+            eu_region:
+                type: boolean
```

Comment:
nit: I wonder if **in the future** we should have a Segment region select input field instead with US (default) and EU as options as it seems they may expand regional segments beyond EU in the future:
> Regional Segment is currently limited to the EU. Future expansion of Regional Segment beyond the EU is under evaluation by Segment Product and R&D.

https://segment.com/docs/guides/regional-segment/#known-limitations

Not something to do right now but to keep in mind in the future.